blockwork/build/caching.py

```diff
@@ -86,6 +205,17 @@ def hash_content(path: Path) -> str:
                 content_hash = hashlib.file_digest(f, 'md5')
         return content_hash.hexdigest()
 
+    @staticmethod
+    def hash_imported_package(package: str) -> str:
```

Comment:
Perhaps just this bit needs lru_cache?

Reply:
This will just read from the `hash_map` dict if the value is already calculated so shouldn't be duplicating work